pkg/operator/controller/certificate/controller.go

```diff
@@ -103,12 +103,13 @@ func (r *reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 			}
 		}
 	}
-	return reconcile.Result{}, utilerrors.NewAggregate(errs)
-}
 
-func routerDeploymentName(ci *ingressv1alpha1.ClusterIngress) types.NamespacedName {
-	return types.NamespacedName{
-		Namespace: "openshift-ingress",
-		Name:      "router-" + ci.Name,
+	ingresses := &ingressv1alpha1.ClusterIngressList{}
+	if err := r.client.List(context.TODO(), &client.ListOptions{Namespace: r.operatorNamespace}, ingresses); err != nil {
+		errs = append(errs, fmt.Errorf("failed to list clusteringresses: %v", err))
+	} else if err := r.ensureRouterCAConfigMap(ca, ingresses.Items); err != nil {
+		errs = append(errs, fmt.Errorf("failed to publish default CA cert: %v", err))
```

Comment:
We only have one CA certificate, I've been calling it the "router CA" certificate.  Granted, it is the CA for default (operator-generated) default certificates, but I think it would be clearer and more consistent to talk about the "router CA".

Reply:
Fixed